src/components/IntersectionDetails/IntersectionDetails.css

```diff
@@ -46,3 +58,17 @@
 .likelihood-1 {
   opacity: 0.45;
 }
+
+/* medium-sized displays and larger */
+@media (min-width: 640px) {
+  aside#intersection-details {
+    flex: 0 0 330px;
+    font-size: 1rem;
+  }
+  .intersection-details {
+    padding: 1.25em;
+  }
+  .intersection-details p {
+    line-height: 1.6;
+  }
+}
```

Comment:
Good catch! Thanks for refactoring, untangling and putting everything in the place it belongs.